### PR TITLE
Feat: Added cart management page

### PR DESCRIPTION
**Pull request by bmendhe23**
##Description
Feat:
1. Added cart management page to the store website.

##Screenshots
ss

##Platforms tested
[✔️] Desktop
[❌] Tablet
[❌] Mobile

### CODE REVIEW

**Review by sakshi006 — APPROVED** [review]
Clean code. Looks good to me!

File: Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+.horizontal-card .horizontal-card-info {
+    padding: 2rem;
+    flex-shrink: 1;
+    align-items: flex-start;
+    justify-content: flex-start;
+}
+
+.horizontal-card .card-img {
+    max-width: 40%;
```
Comment (raieegulhane) [inline Cart/cart.css]: this can be replaced by width: 40%

File: Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    min-width: 40%;
+}
+
+.horizontal-card .card-img img {
+    border-top-left-radius: 0;
```
Comment (raieegulhane) [inline Cart/cart.css]: Shorthand can be used here

**Review by raieegulhane — COMMENTED** [review]
Please take a look at suggested changes. Everything else good to go.

File: Cart/cart.css (hunk 1 above)
Comment (bmendhe23) [reply Cart/cart.css]: did the suggested change. thank you @raieegulhane

File: Cart/cart.css (hunk 2 above)
Comment (bmendhe23) [reply Cart/cart.css]: did the suggested changes. thank you @raieegulhane

**Pull request merged**